platform/darwin/src/MGLFeature.h

```diff
@@ -50,7 +50,7 @@ NS_ASSUME_NONNULL_BEGIN
  <a href="https://www.mapbox.com/vector-tiles/mapbox-streets/">Mapbox Streets</a>
  layer reference.
```

Comment:
Point out that you can change this property and `attributes` on an MGLFeature obtained using feature querying, but doing so has no effect.